packages/amplify_datastore/example/ios/amplify_datastore_unit_tests/resources/SchemaData.swift

```diff
@@ -0,0 +1,81 @@
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import XCTest
+import Amplify
+import Combine
+@testable import AmplifyPlugins
+@testable import amplify_datastore
+
+struct SchemaData {
+    
+    static var PostSchema : ModelSchema = ModelSchema(
+        name: "Post",
+        pluralName: "Posts",
+        fields: [
+        "id" : ModelField(name: "id", type: .string, isRequired: true, isArray: false),
+        "title" : ModelField(name: "title", type: .string, isRequired: true, isArray: false),
+        "created" : ModelField(name: "created", type: .dateTime, isRequired: false, isArray: false),
+        "rating" : ModelField(name: "rating", type: .int, isRequired: false, isArray: false),
+        "blog" : ModelField(name: "blog", type: .model(name: "Blog"), isRequired: false, isArray: false, association: ModelAssociation.belongsTo(targetName: "blogID")),
+        "comments" : ModelField(name: "comments", type: .collection(of: "Comment"), isRequired: false, isArray: true, association: ModelAssociation.hasMany(associatedFieldName: "post") )
+        ]
+    )
+    
+    static var CommentSchema : ModelSchema = ModelSchema(
+        name: "Comment",
+        pluralName: "Comments",
+        fields: [
+            "id" : ModelField(name: "id", type: .string, isRequired: true, isArray: false),
+            "post" : ModelField(name: "post", type: .model(name: "Post"), isRequired: false, isArray: true, association: ModelAssociation.belongsTo(targetName: "postID")),
+            "content" : ModelField(name: "content", type: .string, isRequired: true, isArray: false)
+        ]
+    )
+
+    
+    static var BlogSchema : ModelSchema = ModelSchema(
+            name: "Blog",
+            pluralName: "Blogs",
+            fields: [
+                "id" : ModelField(name: "id", type: .string, isRequired: true, isArray: false),
+                "name" : ModelField(name: "name", type: .string, isRequired: true, isArray: false),
+                "posts" : ModelField(name: "posts", type: .collection(of: "Post"), isRequired: false, isArray: true, association: ModelAssociation.hasMany(associatedFieldName: "blog")
+                )
+            ]
+        )
+    
+    static var PostAuthComplexSchema : ModelSchema = ModelSchema(
+        name: "PostAuthComplex",
+        pluralName: "PostAuthComplexes",
+        authRules: [
+            AuthRule(
+                allow: .owner,
+                ownerField: "owner",
+                identityClaim: "cognito:username"
+                /*
+                operations: [
+                    .read, .delete, .update, .create
+                ]
+                */
```

Comment:
we are not testing operations here?

Reply:
I updated our testing and equatable framework to allow for this to be properly compared.  Since the list order was arbitrary I needed to add extensions to allow for sorting that array and making the individual model operations comparable.